addons/smallarms/CfgMagazines.hpp

```diff
@@ -4,107 +4,106 @@ class CfgMagazines {
     // Magazine updates:
     // 1. Update all magazines with tracer mix to use 1 in 5 mix. Full tracer mags should not be changed!
     // 2. Remove tracers at bottom of magazine.
-    // 3. Do string updates.
 
     // 6.5mm //////////////////////////////////////////
 
     class 30Rnd_65x39_caseless_mag : CA_Magazine { // MX!!
-        displayname = "6.5mm 30Rnd Mag";
+        displayname = CSTRING(30Rnd_65_Name);
         displaynameshort = "6.5mm";
 
         tracersEvery = 0;
         lastRoundsTracer = 0;
     };
 
     class 30Rnd_65x39_caseless_mag_Tracer : 30Rnd_65x39_caseless_mag { // MX!!
-        displayname = "6.5mm 30Rnd Tracer Mag";
+        displayname = CSTRING(30Rnd_Tracer_MX_Name);
         displaynameshort = "6.5mm Tracer";
     };
 
     class 30Rnd_65x39_caseless_green : 30Rnd_65x39_caseless_mag { // Katiba!!
-        displayname = "6.5mm 30Rnd Mag";
+        displayname = CSTRING(30Rnd_65_Name);
         displaynameshort = "6.5mm";
 
         tracersEvery = 0;
         lastRoundsTracer = 0;
     };
 
     class 30Rnd_65x39_caseless_green_Tracer : 30Rnd_65x39_caseless_green { // Katiba!!
-        displayname = "6.5mm 30Rnd Tracer Magazine";
+        displayname = CSTRING(30Rnd_Tracer_Katiba_Name);
         displaynameshort = "6.5mm Tracer";
     };
 
     class 100Rnd_65x39_caseless_mag : CA_Magazine {
-        displayname = "6.5mm 100Rnd Mag";
+        displayname = CSTRING(100Rnd_MX_Name);
```

Comment:
I'm not sure why we are even changing these names??
vanilla is `displayName = "6.5 mm 100Rnd Sand Mag";`
and vanilla also has this mag which we don't touch
`100Rnd_65x39_caseless_khaki_mag { displayName = "6.5 mm 100Rnd Khaki Mag";`

I think it would be better to not touch any of these stings unless we have a reason to

Reply:
Most of the display names have been removed with the exception of those what have differing ammo counts now.